### PR TITLE
Fix local remix-ide url in docs

### PR DESCRIPTION
**Pull request by leonardoalt**
Fixes https://github.com/ethereum/solidity/issues/4855

### CODE REVIEW

File: docs/installing-solidity.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ Remix
 
 `Access Remix online <https://remix.ethereum.org/>`_, you don't need to install anything.
 If you want to use it without connection to the Internet, go to
-https://github.com/ethereum/browser-solidity/tree/gh-pages and download the .ZIP file as
```
Comment (AndriiSuh) [inline docs/installing-solidity.rst:25]: [sha256sum.txt](https://github.com/ethereum/solidity/files/2317475/sha256sum.txt)

**Review by AndriiSuh — CHANGES_REQUESTED** [review]

**Comment by axic** [comment]
@yann300 it seems that `gh-pages` is not updated anymore? Last commit is 6 months ago.

File: docs/installing-solidity.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ Remix
...
+https://github.com/ethereum/remix-ide/tree/gh-pages and download the .ZIP file as
```
Comment (yann300) [inline docs/installing-solidity.rst]: The branch should be `remix_live` ( https://github.com/ethereum/remix-live/tree/gh-pages )
would it be possible to add a link to the release page?
Each release contains a little section on how to run it locally 
https://github.com/ethereum/remix-ide/releases

Comment (axic) [reply docs/installing-solidity.rst]: Could you create a new PR with proper updates to this file?

Comment (chriseth) [reply docs/installing-solidity.rst]: We used to have a version that you could just download and run in a browser - does this not exist anymore?

**Pull request merged**